.github/workflows/ci-tests.yml

```diff
@@ -101,10 +101,11 @@ jobs:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
-        pandas-version: ["1.3.0", "1.5.2", "2.0.1"]
+        pandas-version: ["1.5.3", "2.0.3"]
+        pydantic-version: ["1.10.11", "2.0.2"]
```

Comment:
There were some good bugfixes in pydnatic 2.1.0 (https://docs.pydantic.dev/latest/changelog/#v210-2023-07-25), should we bump this to the latest release (2.1.1) to incorporate that?